index.js

```diff
@@ -31,7 +39,14 @@ app.post('/expenditures', (req, res) => {
     .into('expenditures')
     .returning('*')
     .then((data) => {
-      res.status(201).json(data)
+      // data returns an array with an object of properties and values based on our form submission
+      getCategoryName(data[0].category_id) // returns an object with the id and category name
+        .then((categoryName) => {
+          const obj = data[0]
```

Comment:
Try to use nouns from your app, is this an `expense`? Also, I think right off the bat you can do:

```js
const expense = Object.assign({}, data[0])
```